app/src/main/java/com/conupods/OutdoorMaps/View/MapsActivity.java

```diff
@@ -30,32 +31,28 @@
 import com.google.android.gms.maps.GoogleMap;
 import com.google.android.gms.maps.OnMapReadyCallback;
 import com.google.android.gms.maps.SupportMapFragment;
-import com.google.android.gms.tasks.OnFailureListener;
-import com.google.android.gms.tasks.OnSuccessListener;
 import com.google.android.gms.tasks.Task;
 
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
 public class MapsActivity extends FragmentActivity implements OnMapReadyCallback {
 
+    private static final Logger LOGGER = Logger.getLogger(MapsActivity.class.getName());
+
     private static final String TAG = "MapsActivity";
 
     public static final int LOCATION_PERMISSION_REQUEST_CODE = 1234;
     public static final int RESOLVABLE_API_ERROR_REQUEST_CODE = 51;
 
-    private GoogleMap mMap;
```

Comment:
I still believe mMap should stay as an instance variable. I'm open to discussion though.

Reply:
the mMap private field was always being assigned a value inside the classe's onMapReady() method before being read and wasnt used anywhere else in the class anyway. Making it a local variable in the method avoids confusion.